noxfile.py

```diff
@@ -90,6 +92,12 @@ def unit(session):
     default(session)
 
 
+@nox.session(python=UNIT_TEST_PYTHON_VERSIONS[-1])
+def unit_noextras(session):
```

Comment:
Presumably using just the latest version is fine?
(the CI will finish faster)

On rare occasions an extra requires a more recent Python version, but I can't recall any cases when an extra would be restricted to _older_ Python versions.

Reply:
Sounds good to me. We could also use `DEFAULT_PYTHON_VERSION` here, though that's not always the latest.